### ISSUE TITLE
Separate mini boxes for Montreal and Quebec

### ISSUE THREAD

**Issue by mschoettle** [jeremymoreau/covid19mtl#55]
Continuation from the discussion in #48:

I think it would be great to have the following data visible at one glance in the miniboxes for Mtl and QC each:

* new cases (also with total cases?)
* new deaths (also with total deaths?)
* hospitalisations (QC only?)
* positivity rate (+ trend?)
* current Rt (+ trend?)
* vaccine doses administered (total + new?)
* maybe: active outbreaks

**Comment by jeremymoreau** [comment]
Something like this?

new/total cases MTL | new/total deaths MTL | new/total hospitalisations MTL | new/total ICU admissions MTL | positivity rate MTL | vaccine doses MTL (we don't have this info?)
---|---|---|---|---|---
new/total cases QC   | new/total deaths QC   | new/total hospitalisations QC    | new/total ICU admissions QC | positivity rate QC | vaccine doses QC (and approx. % pop vaccinated)

I'm thinking we could add the new cases/deaths/hospitalisations as a "+n today" superscript above the total in each mini box.

The hospitalisation data for Montreal is in data_mtl.csv ("hos_cum_tot_n" and "hos_quo_tot_n")
I think It'd also be pretty useful if we could add a % capacity for regular/ICU beds if we're able to find that information anywhere.

For Rt, I don't think we have the numbers for just MTL right? Also couldn't find the nb of vaccine doses for just MTL.
We could maybe combine regular/ICU admissions in the same mini box if you want to add another column of info (so that we can stick with 2x6 for a neater responsive layout).

**Comment by jeremymoreau** [comment]
Actually, you're right that for hospitalisations we only have new and total cumulative data for MTL (and only total active for QC, from "data_qc_manual_data.csv"). Unless I'm missing something?

**Comment by mschoettle** [comment]
Yes, exactly. The new numbers/diff from the previous day could be above or below the total.
It might make sense to separate the two rows with a header (sent you a screenshot by email the other day with a sketch). 
We might also want to add the date it was updated, since MTL and QC could differ (although most of this data comes from INSPQ).

Another metric I'd like to have in there is the 7-day incidence per 100k.

> I think It'd also be pretty useful if we could add a % capacity for regular/ICU beds if we're able to find that information anywhere.

True, I've only heard the number in the Sante Montreal press conference from Mdm Belanger from the CUISS Sud-Ouest (if I remember right). I thought it was 1000 beds but that might be for Quebec in total.

> For Rt, I don't think we have the numbers for just MTL right?

No. I messaged Ryan Imgrund who publishes this for MTL among other cities/proviinces on Twitter to see if there is a way he could make it available in machine-readable format on a regular basis.

> Also couldn't find the nb of vaccine doses for just MTL.

There is a CSV from quebec.ca that contains the #doses per region. I started scraping it recently. See: https://github.com/jeremymoreau/covid19mtl/blob/master/app/data/sources/2020-12-29/data_qc_vaccines.csv

> Actually, you're right that for hospitalisations we only have new and total cumulative data for MTL (and only total active for QC, from "data_qc_manual_data.csv"). Unless I'm missing something?

I think that's it. Unless I am not seeing it, the active hospitalisations cannot be calculated from new and total cumulative data either, right?

**Comment by jeremymoreau** [comment]
I've redesigned the header a bit based on the sketch you sent me (3b327e987daa087b48c93622defbe6d4f2ae01a7) to make it a bit more compact (I'm still one of those whitespace = good people though ;) ). I think it looks nicer too. I'd prefer keeping no text between the tiles as I like how everything below the header is one big grid, but I agree that we could add separate update dates for MTL and QC data in the subtitle, e.g. Last update: 2020-12-29 (MTL), 2020-12-30 (QC)

Desktop:
![127 0 0 1_8050_en(Laptop with HiDPI screen) (1)](https://user-images.githubusercontent.com/7446564/103393967-fb0e1300-4ae2-11eb-805e-967c0f92245c.png)

Mobile:
<img src="https://user-images.githubusercontent.com/7446564/103393977-05c8a800-4ae3-11eb-8907-ff9330aff708.png" width="40%">

Sounds good about the other metrics. Once we have all the data (Rt for MTL is all that's missing right?) I can add the new mini boxes. We can put the 7-day incidence in the new cases box as well. And yes I believe you are correct, there is no way of calculating active hospitalisations (we'd need "nb of discharged patients" in addition to "nb of new cases").
